packages/twenty-server/src/modules/calendar/calendar-event-participant-manager/services/calendar-event-participant.service.ts

```diff
@@ -19,6 +21,7 @@ export class CalendarEventParticipantService {
   public async upsertAndDeleteCalendarEventParticipants(
     participantsToSave: CalendarEventParticipantWithCalendarEventId[],
     participantsToUpdate: CalendarEventParticipantWithCalendarEventId[],
+    workspaceId: string,
```

Comment:
I don't think you have to pass the workspaceId here, you could get it from the scopedWorkspaceContextFactory (see twenty-orm.manager.ts). This would avoid the whole drill down and secure the behavior to avoid proposing where the workspace can be provided. That's being said I'm not 100% sure about this pattern of auto-resolving the workspaceId from the request